gamma_bridge/exceptions.py

```diff
@@ -20,3 +20,23 @@ class GammaConnectionError(GammaException):
     def __init__(self, message=None, *args):
         self.message = "External connection error to the rg-gamification-traking application. {}".format(message)
         super(GammaConnectionError, self).__init__(self.message, *args)
+
+
+class GammaValueError(GammaException, ValueError):
+    """
+    Base class for Gamma ValueError.
+    """
+
+    def __init__(self, message=None, *args):
+        self.message = "Wrong type of value. {}".format(message)
+        super(GammaException, self).__init__(message, *args)
+
+
+class NotBaseStorageError(GammaValueError):
```

Comment:
Probably should be inherited from the `GammaException `?

Reply:
It's a special case of GammaValueError.